docs/connectors.md

```diff
@@ -28,21 +28,28 @@ Source means the connector has the ability to get data from an upstream data sto
 | Connector | Source | Destination | Support |
 |-----------|-------|----|-------------|
 | [Algolia](https://github.com/ConduitIO/conduit-connector-algolia) | |✅ | Conduit |
```

Comment:
This one was moved to `https://github.com/conduitio-labs/conduit-connector-algolia`.
```suggestion
| [Algolia](https://github.com/conduitio-labs/conduit-connector-algolia) | |✅ | Conduit |
```